magento/magento2
# 10414 - Add ability to load block and pages by identifiers

Add ability to load block and pages by identifiers

### Description
Add ability to load block and pages by identifiers

### Fixed Issues (if relevant)

1. magento/magento2#10414: Add ability to load block and pages by identifiers

### Manual testing scenarios

1. Developer is able to load the both cms pages and cms blocks on base of provided identifier. Please check the "\Magento\Cms\Model\PageManagment::getByIdentifier" and "\Magento\Cms\Model\BlockManagment::getByIdentifier" methods for details.

### Contribution checklist
